### DISCUSSION TITLE
Crud-service failing migration from v5.4.2 to v6.9.6

### DISCUSSION THREAD

**Discussion by Malv93** [mia-platform/community#394] [Q&A] [answered]
### What product would you like to leave feedback for?

Marketplace

### What Console version are you using?

Console Version: 1.94.2 - Mia-Platform v12.2.0

### Description

Hi,
I'm migrating the Crud-service from v5.4.2 to v6.9.6 in order to use the import upsert feature.
Unfortunately the microservice does not start, and from the logs it looks like it crashes when it tries to register the first modelName.
I'm attaching the logs: [logs (15).txt](https://github.com/mia-platform/community/files/14309919/logs.15.txt)

Someone has a hint about what could be wrong?
I read the docs and the [changelog,](https://docs.mia-platform.eu/docs/runtime_suite/crud-service/changelog) but I wasn't able to find a solution.
I have many modelNames defined in the project and I would prefer not to delete them to debug this issue.

Thank You for your support

**Comment by ianFar96** [comment]
Maybe it is a matter of the service not being up in time for the readiness and healthiness probes. Check the events of the pod to verify it.

**Reply by Malv93** [reply]
Ok, but how should I try to fix it?

From the events I see:
- Created container crud-service
- Successfully pulled image
- Started container crud-service
- Readiness probe failed: Get "http://10.2.72.22:3000/-/ready": dial tcp 10.2.72.22:3000: connect: connection refused
- Liveness probe failed: Get "http://10.2.72.22:3000/-/healthz": dial tcp 10.2.72.22:3000: connect: connection refused

**Reply by ianFar96** [reply, marked as answer]
If those are all the events you see the probes seem to fail only once for each. See what the failureThreshold for each probe is set to, if it is only one then that's you problem, you should increase it. Also increasing the initialDelay could do the trick.

Example of the deployment.yml
![image](https://github.com/mia-platform/community/assets/30315670/2f01ad44-e174-487f-9a76-d7ebfe5e4a08)

**Reply by Malv93** [reply]
Thank You! 
That was the problem. Increasing the initial delay it worked.
